pkg/util/helper/policy.go

```diff
@@ -139,3 +140,10 @@ func SetReplicaDivisionPreferenceWeighted(placement *policyv1alpha1.Placement) {
 	}
 	placement.ReplicaScheduling.ReplicaDivisionPreference = policyv1alpha1.ReplicaDivisionPreferenceWeighted
 }
+
+// SetDefaultGracePeriodSeconds Set the default value of GracePeriodSeconds in ApplicationFailoverBehavior
```

Comment:
```suggestion
// SetDefaultGracePeriodSeconds sets the default value of GracePeriodSeconds in ApplicationFailoverBehavior
```
Please help update the comment of avove function as well.
I mean this one:
```
// SetReplicaDivisionPreferenceWeighted Set the default value of ReplicaDivisionPreference to Weighted
func SetReplicaDivisionPreferenceWeighted(placement *policyv1alpha1.Placement) {
```

Reply:
Done.